applications/dashboard/js/dashboard.js

```diff
@@ -858,26 +858,26 @@ $(document).on('contentLoad', function(e) {
      * @param element - The scope of the function.
      */
     function navbarHeightInit(element) {
-        var $navbar = $('.js-navbar', element);
+        // var $navbar = $('.js-navbar', element);
```

Comment:
Do we still need to keep this function? We should just delete it if it doesn't do anything.